python/smells/manual-collections-create.py

```diff
@@ -0,0 +1,44 @@
+foo = {}
+
+# ruleid:manual-defaultdict-dict-create
```

Comment:
this is great, can you add just a few negative test cases though?

Reply:
Yup, added a few!